app/controllers/users_controller.rb

```diff
@@ -70,15 +68,15 @@ def statistics
 
   private
 
-  def user_slide_with_paginate(user_id)
+  def user_slide_with_paginate(user_id, slides_per_page = 30)
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not 0) spaces for rails indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)